src/components/shared/Autocomplete.js

```diff
@@ -0,0 +1,157 @@
+// @flow
+
+import React, { Component } from "react";
+import { filter } from "fuzzaldrin-plus";
+import classnames from "classnames";
+import { scrollList } from "../../utils/result-list";
+import "./Autocomplete.css";
+
+import SearchInput from "./SearchInput";
+import ResultList from "./ResultList";
+
+type State = {
+  selectedIndex: number,
+  focused: boolean
+};
+
+type Props = {
+  selectItem: (
+    event: SyntheticKeyboardEvent<HTMLElement>,
+    item: Object
+  ) => void,
+  onSelectedItem: (selectedItem: Object) => void,
+  items: Array<Object>,
+  close: (value: any) => void,
+  inputValue: string,
+  placeholder: string,
+  size: string,
+  onChangeHandler: (queryString: string) => void,
+  children: any
+};
+
+export default class Autocomplete extends Component<Props, State> {
```

Comment:
Uh oh, something is up with this diff. Something may have gone wrong while merging in master from upstream.